app/src/main/java/com/futurice/freesound/feature/common/ui/adapter/RecyclerViewAdapter.java

```diff
@@ -121,7 +121,7 @@ private void updateAllItems(@NonNull final List<DisplayableItem> items) {
     }
 
     /**
-     * Do not use for first update of the adapter.
+     * Only used for first update of the adapter.
```

Comment:
Gah, I'll fix it. My intent in changing it was to word in a way that didn't imply that it was part of API, but just the mechanism - so to describe more how it was being used. I'll correct it.